s3bench.py

```diff
@@ -96,6 +100,9 @@ def generate_object_name(cls):
         """This function generates randomized object name"""
         return str(uuid.uuid4())
 
+    def add_prefix(self, obj_name, obj_size):
```

Comment:
We shouldn't use this method due to the fact the addition for the prefix is part of the object path and will be added in the `generate_object_name` function.

Reply:
seems reasonable